Assets/Scripts/PlayerIcon.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using UnityEngine;
+using System.Collections.Generic;
+using UnityEngine.UI;
+
+/// <summary>
+/// Manages Player Icon Location
+/// Appears when camera switch to map mode and when in isometric view
+/// 
+/// </summary>
+public class PlayerIcon : MonoBehaviour
+{
+    public Transform player;
+    private static bool _isoActive;
+    private static bool _mapActive;
```

Comment:
Are these relevant to the player icon? Is this functionally superior to using _iconActive instead?

Reply:
In other words, does this class need to have more information than "do I need to be active or not"?